docs/monitoring/metrics.md

```diff
@@ -335,6 +345,21 @@ Parameters:
 - `ttl` - time-to-live for transmitted UDP packets
 - `addressingMode` - UDP addressing mode to use (UNICAST/MULTICAST)
 
+Example configuration:
+
+{% highlight yaml %}
+
+metrics.reporters: gang
```

Comment:
I think that's very easy to understand when reading the first, but this is definitely good enough! Feel free to merge as is, but can you double check that the examples are actually correct (no typos, etc.) and consistent with the other parts of the documentation.